### ISSUE TITLE
dev_ralph migrate throws an error

### ISSUE THREAD

**Issue by m-ketan** [allegro/ralph#3626]
I'm trying to setup the Ralph's development environment on my mac (osx 11.2.2, m1 chip) but the command ```dev_ralph migrate``` is failing with the below output:

```/Users/uname/Documents/ralph/ralph_env/lib/python3.9/site-packages/django/contrib/contenttypes/models.py:161: RemovedInDjango19Warning: Model class django.contrib.contenttypes.models.ContentType doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class ContentType(models.Model):

/Users/uname/Documents/ralph/ralph_env/lib/python3.9/site-packages/django/contrib/admin/models.py:28: RemovedInDjango19Warning: Model class django.contrib.admin.models.LogEntry doesn't declare an explicit app_label and either isn't in an application in INSTALLED_APPS or else was imported before its application was loaded. This will no longer be supported in Django 1.9.
  class LogEntry(models.Model):

Traceback (most recent call last):
  File "/Users/uname/Documents/ralph/ralph_env/bin/dev_ralph", line 33, in <module>
    sys.exit(load_entry_point('ralph', 'console_scripts', 'dev_ralph')())
  File "/Users/uname/Documents/ralph/src/ralph/__main__.py", line 18, in dev
    main('ralph.settings.dev')
  File "/Users/uname/Documents/ralph/src/ralph/__main__.py", line 14, in main
    execute_from_command_line(sys.argv)
  File "/Users/uname/Documents/ralph/ralph_env/lib/python3.9/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
    utility.execute()
  File "/Users/uname/Documents/ralph/ralph_env/lib/python3.9/site-packages/django/core/management/__init__.py", line 328, in execute
    django.setup()
  File "/Users/uname/Documents/ralph/ralph_env/lib/python3.9/site-packages/django/__init__.py", line 18, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/Users/uname/Documents/ralph/ralph_env/lib/python3.9/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/Users/uname/Documents/ralph/ralph_env/lib/python3.9/site-packages/django/apps/config.py", line 86, in create
    module = import_module(entry)
  File "/opt/homebrew/Cellar/python@3.9/3.9.4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/Users/uname/Documents/ralph/src/ralph/admin/__init__.py", line 2, in <module>
    from ralph.admin.mixins import (
  File "/Users/uname/Documents/ralph/src/ralph/admin/mixins.py", line 23, in <module>
    from mptt.admin import MPTTAdminForm, MPTTModelAdmin
  File "/Users/uname/Documents/ralph/ralph_env/lib/python3.9/site-packages/mptt/admin.py", line 26, in <module>
    from mptt.models import MPTTModel, TreeForeignKey
  File "/Users/uname/Documents/ralph/ralph_env/lib/python3.9/site-packages/mptt/models.py", line 377, in <module>
    class MPTTModel(six.with_metaclass(MPTTModelBase, models.Model)):
RuntimeError: __class__ not set defining 'MPTTModel' as <class 'mptt.models.MPTTModel'>. Was __classcell__ propagated to type.__new__?```

**Comment by vi4m** [comment]
We've tested Ralph on Python 3.6. Try to install it and re-run virtualenv.

**Comment by m-ketan** [comment]
> We've tested Ralph on Python 3.6. Try to install it and re-run virtualenv.

I'm using python 3.9. Is that likely to cause issues? But I'll try with 3.6 anyways.

**Comment by vi4m** [comment]
Yes. You can use `pyenv` to easily install Python 3.6.

**Comment by m-ketan** [comment]
> Yes. You can use `pyenv` to easily install Python 3.6.

Also, can I please ask which versions of node and npm are supported?

**Comment by m-ketan** [comment]
@vi4m Ok so tried with python 3.6.0 and still running into dependency issues:

```
Collecting python-openstackclient>=2.1.0
  Downloading python_openstackclient-5.5.0-py3-none-any.whl (929 kB)
     |████████████████████████████████| 929 kB 16.3 MB/s
INFO: pip is looking at multiple versions of python-novaclient to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of python-keystoneclient to determine which version is compatible with other requirements. This could take a while.
INFO: pip is looking at multiple versions of ralph to determine which version is compatible with other requirements. This could take a while.
ERROR: Cannot install -r requirements/openstack.txt (line 3), -r requirements/openstack.txt (line 4), -r requirements/openstack.txt (line 5) and keystoneauth1==2.3.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested keystoneauth1==2.3.0
    python-keystoneclient 2.3.0 depends on keystoneauth1>=2.1.0
    python-novaclient 3.2.0 depends on keystoneauth1>=2.1.0
    python-ironicclient 1.7.1 depends on keystoneauth1>=2.10.0

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

**Comment by m-ketan** [comment]
Fixed after using python 3.6.0. Closing the issue!
